package.json

```diff
@@ -33,6 +33,7 @@
 		"dist/"
 	],
 	"dependencies": {
+		"@babel/code-frame": "^7.22.13",
```

Comment:
This package is used in [prerendering](https://github.com/preactjs/preset-vite/blob/main/src/prerender.ts#L7).